runtime/autoload/remote/host.vim

```diff
@@ -118,11 +123,20 @@ endfunction
 
 function! remote#host#LoadRemotePlugins() abort
   if filereadable(s:remote_plugins_manifest)
+        \ && !s:loaded_remote_plugins
     exe 'source '.s:remote_plugins_manifest
+    let s:loaded_remote_plugins = 1
   endif
 endfunction
 
 
+function! remote#host#LoadRemotePluginsEvent(event, pattern) abort
+  autocmd! nvim-rplugin
+  call remote#host#LoadRemotePlugins()
+  execute 'doautocmd' a:event a:pattern
```

Comment:
@Shougo I see that `remote#define#FunctionOnHost` defines a FuncUndefined autocmd.